labs/docker-compose.yml

```diff
@@ -38,10 +41,11 @@ services:
 
   connect:
     image: confluentinc/cp-kafka-connect:5.0.0
-    container_name: connect
-    restart: always
+    hostname: connect
     ports:
       - "8083:8083"
+    container_name: connect
```

Comment:
this is not needed. can be removed

Reply:
* `launch.json` and `task.json` are created after you are prompted.  I'd prefer keep them to save the students a click.  There's a debate [here](https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control) that says most projects would include these in source control.
* `container_name` properties I'll remove.  I didn't add any in this update - but they aren't needed.
* `ports:` yes ports aren't needed for most containers.  I'll keep the ports for C3 as our instructions link to `http://localhost:9021` which works both for those in Ubuntu *and* those working in containers locally.